etc/adminhtml/system.xml

```diff
@@ -2,8 +2,11 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
     <system>
         <section id="payment">
-            <group id="omise" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+            <group id="omise" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1">
                 <label>Omise Payment Gateway</label>
+                <fieldset_css>complex omise-section</fieldset_css>
```

Comment:
@jacekstan seems we don't need this `complex` class here? (I tried to remove but nothing happen. Can you try again?)